operators/pkg/tenant-controller/tenant_controller.go

```diff
@@ -307,6 +311,86 @@ func (r *TenantReconciler) checkValidWorkspaces(ctx context.Context, tn *crownla
 	return tenantExistingWorkspaces, workspaces, err
 }
 
+// deleteClusterNamespace deletes the namespace for the tenant, if it fails then it returns an error.
+func (r *TenantReconciler) deleteClusterNamespace(ctx context.Context, tn *crownlabsv1alpha2.Tenant, nsName string) (err error) {
+	ns := v1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: nsName}}
+
+	nsErr := utils.EnforceObjectAbsence(ctx, r.Client, &ns, "personal namespace")
```

Comment:
```suggestion
	err := utils.EnforceObjectAbsence(ctx, r.Client, &ns, "personal namespace")
```

Stick to standard names unless there is a good reason.

Reply:
Resolved in [7e792fd](https://github.com/netgroup-polito/CrownLabs/pull/821/commits/7e792fdc75a73db238a647572c4a75dad19ad848)